dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -375,15 +323,14 @@ public SolrQueryResponse search(DataverseRequest dataverseRequest, List<Datavers
             float score = (Float) solrDocument.getFieldValue(SearchFields.RELEVANCE);
             logger.fine("score for " + id + ": " + score);
             String identifier = (String) solrDocument.getFieldValue(SearchFields.IDENTIFIER);
-            String citation = (String) solrDocument.getFieldValue(SearchFields.DATASET_CITATION);
-            String citationPlainHtml = (String) solrDocument.getFieldValue(SearchFields.DATASET_CITATION_HTML);
+            String citation = (String) solrDocument.getFieldValue(SearchFields.DATASET_CITATION + getCurrentLocaleSuffix());
+            String citationPlainHtml = (String) solrDocument.getFieldValue(SearchFields.DATASET_CITATION_HTML + getCurrentLocaleSuffix());
```

Comment:
Probably we would need some kind of fallback when we cant find field with proper suffix (cases like admin added some language but not reindexed every datasets yet). Everywhere in the app we are fallbacking to english version in such cases.